sdk/servicebus/azure-servicebus/tests/test_topic.py

```diff
@@ -91,3 +93,39 @@ def test_topic_by_servicebus_client_list_topics(self, uamqp_transport, *, servic
         topics = client.list_topics()
         assert len(topics) >= 1
         # assert all(isinstance(t, TopicClient) for t in topics)
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @CachedServiceBusTopicPreparer(name_prefix='servicebustest')
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasser()
+    def test_topic_by_servicebus_client_send_large_messages_w_sleep(self, uamqp_transport, *, servicebus_namespace=None, servicebus_topic=None, **kwargs):
+        fully_qualified_namespace = f"{servicebus_namespace.name}{SERVICEBUS_ENDPOINT_SUFFIX}"
+        credential = get_credential()
+        
+        # message of 160 kb - requires multiple transfer frames
+        def create_large_message():
+            size = 160
+            large_dict = {
+                "key": "A" * 1024
+            }
+            for i in range(size):
+                large_dict[f"key_{i}"] = "A" * 1024
+            return json.dumps(large_dict)
+
+        body = create_large_message()
+
+        with ServiceBusClient(
+            fully_qualified_namespace=fully_qualified_namespace,
+            credential=credential,
+            logging_enable=False,
+            uamqp_transport=uamqp_transport
+        ) as sb_client:
+            with sb_client.get_topic_sender(servicebus_topic.name) as sender:
```

Comment:
This issue doesn't repro unless we turn on/add logging here, due to some strange threading behavior. Adding in the logging here to make sure this bug is being hit.